internal/gcloud/gcloud_test.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package gcloud_test
+
+import (
+	"bytes"
+	"context"
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/cloudsql-proxy/v2/internal/gcloud"
+	exec "golang.org/x/sys/execabs"
+)
+
+func TestGcloud(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping gcloud integration tests")
+	}
+
+	configureGcloud := func(t *testing.T) func() {
+		// The following configures gcloud using only GOOGLE_APPLICATION_CREDENTIALS.
+		dir, err := ioutil.TempDir("", "cloudsdk*")
+		if err != nil {
+			t.Fatalf("failed to create temp dir: %v", err)
+		}
+		os.Setenv("CLOUDSDK_CONFIG", dir)
```

Comment:
Can we beef up this comment a bit - what does "CLOUDSDK_CONFIG" have to do with any of it?

Reply:
Done.